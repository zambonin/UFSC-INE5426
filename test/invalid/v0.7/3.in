int a
float b
bool fun f(int x, int y)
bool fun f(int x, int y){
ret false	
}
bool c
c = f(a, b)
