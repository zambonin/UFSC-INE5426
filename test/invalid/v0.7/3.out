[Line 8] semantic error: parameter y expected integer but received float
